app.py

```diff
@@ -41,7 +49,7 @@ def get_twitch_stream(user):
     return json.loads(r.text)
 
 
-def build_streamer_json(user, stream, participant_id):
+def build_streamer_json(user, stream, participant_id, user_info):
```

Comment:
Does the `user_info` object also contain the same information we pass in for `user` and `participant_id`? If so, can we do away with `user` and `participant_id` and just pull those details of the `user_info` object?